examples/storm-pmml-examples/src/main/resources/KNIME_PMML_4.1_Examples_single_audit_logreg.xml

```diff
@@ -0,0 +1,259 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we know what the license of this file is? It wasn't clear to me from the dmg.org website.